css/css-text/white-space/pre-wrap-008.html

```diff
@@ -1,23 +1,22 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>CSS Text level 3 Test: preserved white space at the end of and white-space:pre-wrap is wrapped when overflow-wrap is break-spaces</title>
+<title>CSS Text level 3 Test: preserved white space at the end of line and white-space is break-spaces</title>
 <link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
 <link rel="help" href="https://drafts.csswg.org/css-text-3/#white-space-phase-2">
-<link rel="help" href="https://drafts.csswg.org/css-text-3/#valdef-overflow-wrap-break-spaces">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#valdef-white-space-break-spaces">
 <link rel="match" href="reference/pre-wrap-001-ref.html">
-<meta name="assert" content="preserved white space at the end of the line is wrapped when the white-space property is set to pre-wrap and overflow-wrap is break-spaces.">
+<meta name="assert" content="preserved white space at the end of the line is wrapped when the white-space property is set to break-spaces.">
 <style>
 div {
   font-size: 20px;
   font-family: Ahem;
   line-height: 1em;
-  white-space: pre-wrap;
+  white-space: break-spaces;
```

Comment:
it's weird that this test is named pre-wrap-xxx but itsn't testing such value anymore.

Reply:
good point. I'll rename.